bindings/rust-bindings/src/superchain.rs

```diff
@@ -2,11 +2,23 @@
 
 use super::{Chain, ChainConfig, ChainList, HashMap, RollupConfigs, SuperchainConfig};
 
-/// A list of chain configs.
+/// A list of Hydrated Superchain Configs.
 #[derive(Debug, Clone, Default, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
 #[serde(rename_all = "camelCase")]
-pub struct ChainConfigList {
-    /// Chain configs.
+pub struct HydratedSuperchainConfigs {
+    /// A list of hydrated superchain configs.
+    pub superchains: Vec<HydratedSuperchainConfig>,
+}
+
+/// A hydrated Superchain Config.
+#[derive(Debug, Clone, Default, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct HydratedSuperchainConfig {
```

Comment:
Is there a non-hydrated `SuperchainConfig` type I'm missing? If not, would pref we drop the `Hydrated` prefix to the type names.

Reply:
There is, see right below on line 20